GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/CenterpieceViewModel.cs

```diff
@@ -40,7 +41,7 @@ public CenterpieceViewModel()
             AllUsers.Add(GlobalData.GetInstance().EarthUser);
         }
 
-        private void AllUsersCollectionChanged(object sender, NotifyCollectionChangedEventArgs changedEventArgs)
+        public void AllUsersCollectionChanged(object sender, NotifyCollectionChangedEventArgs changedEventArgs)
```

Comment:
Instead of making all of these functions `public` for the sake of testing, it may be a better idea to find ways to trigger the functions they are meant to be triggered through existing public interfaces. 

It's not a huge deal, but I think it makes the code more readable to only publicly expose code that is meant to be publicly exposed

Reply:
Good catch. I was a bit unsure on this, but I think the test code is worth another pass, considering.